composer.json

```diff
@@ -12,7 +12,7 @@
 	],
 	"support": {
 		"email": "adi-ia@lists.wisc.edu",
-		"source": "https://bitbucket.org/uwmadison_doit/caos-php-client"
+		"source": "https://github.com/cknuth/caos-php-client"
```

Comment:
I think your rebase went awry. The values deleted here are the ones that are correct.